README.md

```diff
@@ -1,41 +1,67 @@
+
+This library is a command line tool for interacting with the FT.com CDN, [Fastly](https://www.fastly.com/).
+
+
+## Requirements
+
+* Node 5.x
 
-Command Line Utility for interacting with fastly
 
 ## Installation
 
-```
-npm install --save-dev @financial-times/fastly-tools
+```sh
+git clone git@github.com:Financial-Times/fastly-tools.git
+cd fastly-tools
+make install
 ```
 
-## Usage
 
-	fastly [options] [command]
+## Development
 
-      Commands:
+### Testing
 
-        deploy [options] [folder]
-           Deploys VCL in [folder] to the specified fastly service.  Requires FASTLY_APIKEY env var
+In order to run the tests locally you'll need to run:
 
-      Options:
+```sh
+make test
+```
 
-        -h, --help  output usage information
+### Install from NPM
 
+```sh
+npm install --save-dev @financial-times/fastly-tools
+```
+
+### Usage
 
-## Commands
+```sh
+Usage: fastly [options] [command]
 
-### deploy
-Deploys VCL in [folder] to the specified fastly service.  Requires FASTLY_APIKEY env var
+Options:
+  -h, --help                 output usage information
+
+Commands:
+  deploy [options] [folder]  Deploys VCL in [folder] to the specified fastly service.  Requires FASTLY_KEY env var
+```
 
-  Usage: deploy [options] [folder]
+### Deploy
 
-  Options:
+Deploys VCL in [folder] to the specified fastly service. Requires FASTLY_APIKEY environment variable.
 
-    -h, --help                output usage information
-    -m, --main <main>         Set the name of the main vcl file (the entry point).  Defaults to "main.vcl"
-    -v, --vars <vars>         A way of injecting environment vars into the VCL.  So if you pass --vars AUTH_KEY,FOO the values ${AUTH_KEY} and ${FOO} in the vcl will be replaced with the values of the environment variables.  If you include SERVICEID it will be populated with the current --service option
-    -e, --env                 Load environment variables from local .env file (use when deploying from a local machine
-    -s, --service <service>   REQUIRED.  The ID of the fastly service to deploy to.
-    -V --verbose              Verbose log output
-    -b --backends <backends>  Upload the backends specified in <backends> via the api
-    -k --api-keys <keys>      list of alternate api keys to try should the key stored in process.env.FASTLY_API_KEY hit its rate limit
+```sh
+Usage: deploy [options] [folder]
+
+Deploys VCL in [folder] to the specified fastly service.  Requires FASTLY_KEY env var
+
+Options:
+  -m, --main <main>               Set the name of the main vcl file (the entry point).  Defaults to "main.vcl"
+  -v, --vars <vars>               A way of injecting environment vars into the VCL.  So if you pass --vars AUTH_KEY,FOO the values {$AUTH_KEY} and ${FOO} in the vcl will be replaced with the values of the environmemnt variable.  If you include SERVICEID it will be populated with the current --service option
```

Comment:
```suggestion
  -v, --vars <vars>               A way of injecting environment vars into the VCL.  So if you pass --vars AUTH_KEY,FOO the values {$AUTH_KEY} and ${FOO} in the vcl will be replaced with the values of the environment variable.  If you include SERVICEID it will be populated with the current --service option
```

Reply:
This typo is still present.

Reply:
Oops - sorry, it's actually present in another file `bin/fastly-tools.js `, which is a pre-existing line of code, so change it if you feel inclined but it's not a blocker.